examples/astro/src/react/content-types/activity.tsx

```diff
@@ -0,0 +1,40 @@
+import type { FC } from "react";
+import type { DotCMSContentlet } from "../../types";
+import useImageSrc from "../hooks/useImageSrc";
```

Comment:
Astro don't have something similar to image loader?

Reply:
Yes but it only works inside astro components